Makefile

```diff
@@ -3,23 +3,64 @@
 ANOLIS = anolis
 TIDY-HTML5 = tidy-html5
 
+# This hacky set of shell commands should return the URL of the latest
+# version of the spec published in /TR/ space. This URL is used to compute the
+# "Previous version" link automatically.
+# TODO: Rather use the "Web API" that W3C will soon expose once it is available
+PUBLISHED-URL=$(shell curl -s http://www.w3.org/TR/presentation-api/ \
+	| grep "This version" --after-context=6 \
+	| grep "<a href" \
+	| sed "s/^.*<a href=\"//" \
+	| sed "s/\".*//")
+
+
+# Default rule generates the Editor's Draft
 all: tidy index.html xrefs/presentation.json
 
+# Rule to generate the First Public Working Draft
 fpwd: tidy xrefs/presentation.json releases/FPWD.html
 
+# Rule to generate a new Working Draft release
+wd: tidy xrefs/presentation.json releases/WD.html
+
+
 releases/FPWD.html: Overview.src.html ../xref Makefile
 	$(ANOLIS) --w3c-status=WD --w3c-compat-substitutions \
 	--w3c-compat-crazy-substitutions --w3c-shortname="presentation-api" \
 	--omit-optional-tags --quote-attr-values --enable=xspecxref \
 	--xref="../xref" --enable=refs $< $@
 
+
+releases/WD.html: Overview.src.html ../xref Makefile
+	# Generate the Working Draft release from the source spec
+	$(ANOLIS) --w3c-status=WD --w3c-compat-substitutions \
+	--w3c-compat-crazy-substitutions --w3c-shortname="presentation-api" \
+	--omit-optional-tags --quote-attr-values --enable=xspecxref \
+	--xref="../xref" --enable=refs $< $@
+
+	# Generate the "Previous version" link
+	sed -i "s|<!--previous-version-->|\n        <dt>Previous version:</dt>\n        <dd><a href=\"$(PUBLISHED-URL)\">$(PUBLISHED-URL)</a></dd>|" $@
+
+
 index.html: Overview.src.html ../xref Makefile
+	# Make a temp copy of the source spec, replacing [VERSION] used in the
+	# "This version" link with the right GitHub page address
+	# (Anolis would generate a URL in /TR/ space otherwise, which is good
+	# for Working Draft releases but not for Editor's Drafts)
+	sed "s|\[VERSION\]|http://w3c.github.io/presentation-api/|" $< > tmp.html
+
+	# Generate the Editor's Draft from the temp copy
```

Comment:
In theory, yes, as I see that Anolis reads from stdin if the input file is not specified. In practice, I do not manage to make that work: it is as if Anolis missed the first 200 lines of the HTML content sed outputs.

It may be because Anolis wants to read binary from stdin and rather gets text content. I tried to launch Anolis with "Python -u" to put stdin in binary mode. The result is slightly better but still not correct, so there must be something else.  I'm using Windows nowadays, this behavior may be OS-specific. Unless someone knows how to fix that, I suggest to keep the temporary file.